pkg/ruler/api_test.go

```diff
@@ -75,22 +75,46 @@ func makeUserID() string {
 }
 
 // makeRulerConfig makes an arbitrary ruler config
-func makeRulerConfig() configs.RulesConfig {
-	return configs.RulesConfig(map[string]string{
-		"filename.rules": makeString(`
+func makeRulerConfig(rfv configs.RuleFormatVersion) configs.RulesConfig {
+	switch rfv {
+	case configs.RuleFormatV1:
+		return configs.RulesConfig(map[string]string{
+			"filename.rules": makeString(`
+			# Config no. %d.
+			ALERT ScrapeFailed
```

Comment:
Could we have the indentation the same between v1 and v2 here?  For consistency, and I think this would also improve the diffs.

Reply:
Done, I outdented this to what it used to be.